rfcs/20200205-standalone-keras-repository.md

```diff
@@ -0,0 +1,394 @@
+# Standalone Keras Repository
+
+| Status        | Proposed |
+:-------------- |:---------------------------------------------------- |
+| **RFC #**     | [202](https://github.com/tensorflow/community/pull/202) |
+| **Author(s)** | Qianli Zhu (scottzhu@google.com), Francois Chollet (fchollet@google.com) |
+| **Sponsor**   | Karmel Allison (karmel@google.com) |
+| **Updated**   | 2020-02-05                         |
+
+## Objective
+
+Move the Keras code from the TensorFlow main GitHub repository to its own
+repository, with TensorFlow as a dependency.
+
+## Motivation
+
+### Build times
+
+Building the open-source TensorFlow project end-to-end is an extensive exercise. 
+With a standard GCP instance, it might take more than one hour to finish the whole
+build process (it might take longer with a Mac laptop). Although the local build 
+cache might help speed up the follow-up builds, the initial time cost is too 
+high for regular software development workflows. Internally, Google has a
+distributed build and caching service, which Googlers heavily rely on,
+that can build TensorFlow and run all Keras tests within 5 mins. Sadly,
+we can't expose this to external contributors.
+
+Currently, any contribution to Keras code will require building all of
+TensorFlow, which is quite expensive to do for average users.
+Having a separate repository will allow the Keras package to be built
+without building TensorFlow. This should greatly improve the 
+velocity of open-source developers when they contribute to Keras code.
+
+### Community Benefit
```

Comment:
Isn't this the same as the build time section above?

Reply:
Done. Updated.